sonar-core/src/main/java/org/sonar/core/util/stream/MoreCollectors.java

```diff
@@ -110,6 +112,13 @@ private MoreCollectors() {
       ImmutableSet::copyOf);
   }
 
+  /**
+   * A Collector into an {@link EnumSet} of specified enumeration.
+   */
+  public static <E extends Enum<E>> Collector<E, ?, EnumSet<E>> toEnumSet(Class<E> enumClass) {
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Remove usage of generic wildcard type. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1452)